webserver/templates/navbar.html

```diff
@@ -27,11 +27,7 @@
       {# Not showing user stuff on error pages. If attempt to load user info fails there, page will not render. #}
       {% if not error %}
         <ul class="nav navbar-nav navbar-right">
-          {% if not current_user or current_user.is_anonymous() %}
           <li class="import-link"><a href="{{ url_for('index.import_data') }}">Import!</a></li>
-          {% else %}
```

Comment:
What does this do? Always send you to import, which redirects if you're logged in?